src/60_Samples_%26_Templates/Housing.html

```diff
@@ -202,41 +233,41 @@ <h4>Floorplan</h4>
         </section>
         <section>
           <h4>Mortgage Calculator</h4>
```

Comment:
change h4 to h2 for semantic consistency.

Reply:
done